db/migrations/20220918104526-create-users-table.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      first_name: {
+        type: Sequelize.STRING,
+      },
+      last_name: {
+        type: Sequelize.STRING,
+      },
```

Comment:
I think we can remove first name and last name since we want to address the customer/seller  by their username (creator name)